modules/show_recent_added.js

```diff
@@ -0,0 +1,48 @@
+let recentChecked;
+
+// Scroll to show recent added books and highlights them when clicking link1
+function scrollShowRecentAdded(allRecent = document.querySelectorAll('.recent-added')) {
```

Comment:
- [ ] It would be better to use arrow functions here too, as this is a better practice in ES6, please do this for all the functions.